daemon/api_test.go

```diff
@@ -189,6 +203,21 @@ func (s *apiBaseSuite) TearDownTest(c *check.C) {
 	unsafeReadSnapInfo = unsafeReadSnapInfoImpl
 	ensureStateSoon = ensureStateSoonImpl
 	dirs.SetRootDir("")
+
+	// TODO: do something mockable
+	snapstateCoreInfo = snapstate.CoreInfo
```

Comment:
me confused there is some of this already in the function just above, one or the other should go away :)